src/fuzzfetch/fetch.py

```diff
@@ -1077,26 +1159,37 @@ def extract_build(self, path=".", gtest=False):
                 os.path.join(path, "dependentlibs.list.gtest"),
             )
 
-        if self._flags.coverage:
-            self.extract_zip("code-coverage-gcno.zip", path=path)
+        if have_exec:
+            if self._flags.coverage:
+                self.extract_zip("code-coverage-gcno.zip", path=path)
 
-        if not self._flags.asan and not self._flags.tsan and not self._flags.valgrind:
-            os.mkdir(os.path.join(path, "symbols"))
+            if (
+                not self._flags.asan
+                and not self._flags.tsan
+                and not self._flags.valgrind
+            ):
+                os.mkdir(os.path.join(path, "symbols"))
+                try:
+                    self.extract_zip(
+                        "crashreporter-symbols.zip", path=os.path.join(path, "symbols")
+                    )
+                except FetcherException:
+                    # fuzzing debug builds no longer have crashreporter-symbols.zip
+                    # (bug 1649062)
+                    # we want to maintain support for older builds for now
+                    if not self._flags.fuzzing:
+                        raise
+
+        # any still remaining targets are assumed to be test artifacts
+        for target in targets_remaining:
```

Comment:
Not sure if you want to address it in this PR or not but xpcshell will also require an `xpcshell.fuzzmanagerconf`

Reply:
Yes, I might as well. As we discussed in matrix, `xpcshell` should be extracted to the same path as `firefox` anyways to avoid a startup assertion in debug builds.

Reply:
Created https://github.com/MozillaSecurity/fuzzfetch/issues/96 for this instead. We're using `firefox -xpcshell`, so it's not urgent.